revisions.py

```diff
@@ -606,25 +743,27 @@ def rule_revisions_cleanup_collect(ctx, batch_size):
     if user.user_type(ctx) != 'rodsadmin':
         raise Exception("The revision cleanup jobs can only be started by a rodsadmin user.")
 
-    if has_spool_data(constants.PROC_REVISION_CLEANUP):
-        return "Existing spool data present. Not adding new revision cleanup data."
+    log.write(ctx, '[revisions] Revision cleanup scan job starting.')
+    verbose = verbose_flag == "1"
+    revisions_list = get_spool_data(constants.PROC_REVISION_CLEANUP_SCAN)
+
+    if revisions_list is None:
+        log.write(ctx, '[revisions] Revision cleanup scan job stopping - no more spooled revision scan data.')
```

Comment:
This results in a duplicate '[revisions]' tag in the logging. Module name is also added by logging util.
```
Oct 25 10:36:46 pid:20511 NOTICE: writeLine: inString = {rods#tempZone} [revisions] [revisions] Revision cleanup processing job completed - 4 candidates for 27 versioned data objects (4 successful / 0 errors).
Oct 25 10:36:47 pid:20520 NOTICE: writeLine: inString = {rods#tempZone} [revisions] [revisions] Revision cleanup scan job starting.
Oct 25 10:36:47 pid:20520 NOTICE: writeLine: inString = {rods#tempZone} [revisions] [revisions] Revision cleanup scan job stopping - no more spooled revision scan data.
Oct 25 10:36:47 pid:20529 NOTICE: writeLine: inString = {rods#tempZone} [revisions] [revisions] Revision cleanup job processing starting.
Oct 25 10:36:47 pid:20529 NOTICE: writeLine: inString = {rods#tempZone} [revisions] [revisions] Revision cleanup processing job stopping - no more spooled revision data.
```